src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/CoreExtension.php

```diff
@@ -362,4 +363,20 @@ public function hasPermission($component, $instance, $level)
 
         return (bool) $result;
     }
+    
+    /**
+     * @param string $modname
+     * @param bool $force
+     * @return bool
+     */
+    public function modAvailable($modname = null, $force = false)
+    {
+        if (empty($modname)) {
```

Comment:
so remove the default value, that will force an argument to be set and you won't have to have code to check that it is set.